.eslintrc

```diff
@@ -17,7 +17,7 @@
     "eqeqeq": ["error", "smart"],
     "camelcase": ["error", {"properties": "never"}],
     "dot-notation": "error",
-    "func-names": "error",
+    "func-names": "off",
```

Comment:
Torn on this because I know it's necessary for the agent, but will make writing tests hell...

Reply:
You can put a separate `.eslintrc` file in the `tests` directory with it turned off. That's what we do in the agent: https://github.com/newrelic/node-newrelic/blob/master/test/.eslintrc